Reservaciones/Screens/Login.js

```diff
@@ -1,13 +1,71 @@
-import React from 'react';
+
+import React, { useState, useEffect } from 'react';
 import styleLogin from '../Styles/LoginStyle';
 import { GoogleSocialButton } from "react-native-social-buttons";
-import { View, Text, TextInput, TouchableOpacity, Linking} from 'react-native';
+import { View, Text, TextInput, TouchableOpacity, Linking, Button} from 'react-native';
+import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome'
+import { faEye } from '@fortawesome/free-solid-svg-icons'
+import auth from '@react-native-firebase/auth';
+import { GoogleSignin } from '@react-native-community/google-signin';
 
+GoogleSignin.configure({
+  webClientId: '346661789891-e6taibn68bvqogs5h93gs9bgdbt3utlp.apps.googleusercontent.com',
+});
 
+async function onGoogleButtonPress() {
+   // Get the users ID token
+   const { idToken } = await GoogleSignin.signIn();
+ 
+   // Create a Google credential with the token
+   const googleCredential = auth.GoogleAuthProvider.credential(idToken);
+ 
+   // Sign-in the user with the credential
+   return auth().signInWithCredential(googleCredential);
```

Comment:
Comments and spaces feel unnecessary.
```suggestion
   const { idToken } = await GoogleSignin.signIn();
   const googleCredential = auth.GoogleAuthProvider.credential(idToken);
 
   return auth().signInWithCredential(googleCredential);
```